mizar/networkpolicy/networkpolicy_util.py

```diff
@@ -376,6 +394,29 @@ def add_pod_ip_into_cidrs_map(self, pod, policy_name, pod_ip_set):
             if pod.metadata.name not in networkpolicy_opr.store.pod_names_to_be_ignored_by_networkpolicy:
                 pod_ip_set.append("{}/32".format(pod.status.pod_ip))
 
+    def get_pod_label_values_by_label_dict(self, label_dict):
+        pod_label_values = []
+        query_list = self.get_label_query_list(label_dict)
```

Comment:
This is going to affect arktos mizar integration?

Reply:
No, this won't affect.